kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/flight/Flight.java

```diff
@@ -1,2 +1,60 @@
-package com.kodilla.good.patterns.flight;public class Flight {
+package com.kodilla.good.patterns.flight;
+
+import java.util.Objects;
+
+public class Flight {
+
+    private final String departureAirport;
```

Comment:
Takie zamodelowanie klasy wprowadza usztywnienie - pozwala reprezentować loty tylko z jedną przesiadką.
Można to wyrazić bardziej uniwersalnie, np. jako listę lotnisk składających się na dany lot.